b3lb/rest/b3lb/tasks.py

```diff
@@ -28,6 +28,11 @@
 import json
 from jinja2 import Template
 
+#
+# Celery task routines
+#
+LIMIT_METRICS = [("attendee_limit", "Attendee soft limit"), ("meeting_limit", "Attendee soft limit")]
```

Comment:
meeting_limit:  `Meeting limit`

Reply:
It's an artifact from the development and wasn't used.